docs/modules/ROOT/pages/guides/v3-migration/index.adoc

```diff
@@ -0,0 +1,248 @@
+[[v3-migration]]
+= 3.0.0 Migration
+This document lists all breaking changes from version 2.x.y to 3.0.0 and how to update.
+
+== How to upgrade
+Simply update `@neo4j/graphql` using npm or your package manager of choice:
+
+[source, bash, indent=0]
+----
+npm update @neo4j/graphql
+----
+
+== Asynchronous schema generation
+Schema generation is now asynchronous. Instead of using the property `schema`, now the method `getSchema` will return the schema
+as a `Promise`. This means that creating a server now requires awaiting for that method:
+
+Instead of
+[source, JavaScript, indent=0]
+----
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+const server = new ApolloServer({
+    schema: neoSchema.schema,
+});
+----
+
+Now you'll need to do the following:
+
+[source, JavaScript, indent=0]
+----
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+neoSchema.getSchema().then((schema) => {
+    const server = new ApolloServer({
+        schema: schema
+    });
+});
+----
+
+== Relationship changes
+This release contains an overhaul of our relationship validations, which will require a few changes to the schema.
+
+=== Many-to-* relationships
+To improve consistency and validation, **all** "many-to-*" relationships need to be defined as _required_ in the schema:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    actors: [Actor!]! @relationship(type: "ACTED_IN", direction: IN)
+    director: Director @relationship(type: "DIRECTED", direction: IN)
+}
+----
+
+Note that any other notation, such as `[Actor]` or `[Actor!]` will **not** be valid. "One-to-one" relationships
+such as `Director` remain unchanged.
+
+=== Relationship cardinality
+Runtime checks for "one-to*" relationships have been added in this release, ensuring that the correct number of relationships exist. This means that some
+databases with inconsistent relationships between the schema definition and the actual data may now fail in some queries.
+This may have happened due to different reasons such as:
+* Direct changes in the database.
+* Changes to the Type Definitions, leading to an inconsistent database.
```

Comment:
More a question here but do we want this to be a list with each element on a new line? If not, ignore this comment please. If so then it isn't the case just now: https://neo4j-graphql-docs-977.surge.sh/graphql-manual/3.0/guides/v3-migration/#_relationship_cardinality

Reply:
Good catch. however, due to Darrell's comment this is not going to be a list, so problem solved XD